zubhub_backend/zubhub/activitylog/permissions.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.throttling import UserRateThrottle
+from django.conf import settings
+
+
+class SustainedRateThrottle(UserRateThrottle):
+    scope = 'sustained'
+
+    def allow_request(self, request, view):
+        if settings.DEBUG:
+            return True
+        return super().allow_request(request, view)
```

Comment:
Maybe add a newline at the end of this file. Same applies to other files that currently are missing a newline.

Reply:
Done.